src/base/promptElements.tsx

```diff
@@ -133,6 +133,12 @@ export interface TextChunkProps extends BasePromptElementProps {
 	breakOnWhitespace?: boolean;
 }
 
+export interface ImageProps extends BasePromptElementProps {
+	imageUrl: string;
```

Comment:
We should give some details around how images are expected to be provided. Internally we assume this is the raw base64 image content, but this is kind of unexpected with a plain reading because that's not a URL. Maybe this should also be just called `src` to mirror HTML and avoid the URL assumption.